src/app/pages/RuntimeAccountDetailsPage/index.tsx

```diff
@@ -27,6 +27,7 @@ import {
   transfersContainerId,
 } from '../../utils/tabAnchors'
 import { ParamSetterFunction } from '../../hooks/useTypedSearchParam'
+import { Staking } from './Staking'
 
 export type RuntimeAccountDetailsContext = {
```

Comment:
Total delegated/undelegated balance are not shown. Is that planned as a separate PR, or did some commits get lost?
<img width="419" height="145" alt="image" src="https://github.com/user-attachments/assets/bea6bfa6-2336-47c2-86bd-1b6c7f97fd92" />